src/qibolab/serialize.py

```diff
@@ -41,6 +41,9 @@ def load_qubits(runcard: dict) -> Tuple[QubitMap, CouplerMap, QubitPairMap]:
     objects.
     """
     qubits = {q: Qubit(q, **char) for q, char in runcard["characterization"]["single_qubit"].items()}
+    if runcard["kernel_path"]:
+        for qubit in qubits.values():
+            qubit.kernel_path = Path(runcard["kernel_path"])
```

Comment:
Serialize with:
https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.as_posix
it will be standard, even though not portable.

Deserialize with just `Path`: it should already work on all platforms, and it will ensure the portability.

Reply:
I just saw it now, it looks simpler to what I did, if my thing doesn't work again I will try with this